week03/205. Isomorphic.cpp

```diff
@@ -0,0 +1,33 @@
+//tc = O(n), mc = O(1)
+//n=number of elements
+
+class Solution {
+public:
+
+    unordered_map<char, char> s_map, t_map;
+    bool notIso = false;
+    bool isIso = true;

```

Comment:
Wrong solution. You can't put them as member variable. It does not make any sense. It will surely give wrong answers for multiple invocation of `isIsomorphic()` from same Solution object.

Reply:
I thought maybe it would be more straightforward. But, it turned into wrong decision. Ok... I understand.
But, for my curiosity, I want to know what kind of issue will arise here as it is giving the right answer for this certain solution?